flink-runtime-web/web-dashboard/src/app/pages/overview/statistic/overview-statistic.component.less

```diff
@@ -17,50 +17,56 @@
  */
 @import "theme";
 
-nz-card {
-  ::ng-deep {
-    .ant-card-head {
-      border-bottom: 0;
-    }
+:host {
+  nz-card {
+    ::ng-deep {
+      .ant-card-head {
+        border-bottom: 0;
+      }
 
-    .ant-card-body {
-      padding: 0 24px 8px;
+      .ant-card-body {
+        padding: 0 24px 8px;
+      }
     }
   }
-}
 
-.total {
-  color: @success-color;
-  font-size: 36px;
-  line-height: 38px;
-}
-
-.footer {
-  margin-top: 16px;
-  padding-top: 9px;
-  border-top: 1px solid @border-color-split;
+  .total {
+    color: @success-color;
+    font-size: 36px;
+    line-height: 38px;
+  }
 
-  .field {
-    display: inline-block;
-    font-size: 14px;
-    line-height: 22px;
+  .footer {
+    margin-top: @margin-md;
+    padding-top: @padding-sm;
+    border-top: 1px solid @border-color-split;
```

Comment:
nit: I'd like to use variables like `@border-width-base` .etc. to make the code style uniform :)